README.md

```diff
@@ -395,16 +392,16 @@ java -jar solid-conformance-test-harness-runner.jar
 ### Release
 Update CHANGELOG.md to highlight new features before starting the release.
 ```shell
-mvn release:prepare
+./mvnw release:prepare
```

Comment:
Should you also mention `./mvnw release:perform`?

Reply:
At some point yes but so far this process gets me to a tagged version in github to which I add the built jar. That and the docker image are likely to be the main deployables and I wasn't sure what maven repo it would be deployed to. I didn't want to add that step until I knew what we were doing with it.